src/shared/theme/sizes.ts

```diff
@@ -4,3 +4,18 @@ export function sizes<B extends boolean>(n: number, raw?: B): B extends false ?
 export function sizes(n: number, raw?: boolean) {
   return raw ? base * n : `${base * n}px`
 }
+
+export const zIndex = {
```

Comment:
I think whether maybe we should choose some meaningful names for those. I'm afraid this will become the same problem as with the `xl` sizes. Also, for example the `inifnity` is used for the header, but if at some point we want a full page overlay, we'll need that to have a higher zIndex. For now I think we could use something like `header`, `background`, `localOverlay`. If in the future we would need more, we could then pick other meaningful names. What do you think about that?

Reply:
I can see this being hard to use if the scale has a lot of values, but given that will only ever use a handful of values for a `z-index`, I agree that it would be an improvement over the current version and will change it